doc/content/modules/user-manual/pages/release-notes/2025.4.0.adoc

```diff
@@ -44,6 +44,17 @@ For example importing the following SysML file would now create a valid semantic
 }
 ```
 
+- Improve textual export of a `SuccessionAsUsage` to property handle its name.
+For example exporting a model with a named `SuccessionAsUsage` would produce the following file:
```

Comment:
This example is strange as you don't provide the model (which is impossible in this release note).
So please explain with text what is the input model, or change your example to reduce it to the minimum.